packages/modeling/src/geometries/geom3/transform.js

```diff
@@ -15,10 +13,9 @@ const create = require('./create')
  * let newgeometry = transform(fromXRotation(degToRad(90)), geometry)
  */
 const transform = (matrix, geometry) => {
-  const newgeometry = create(geometry.polygons) // reuse the polygons
-  newgeometry.isRetesselated = geometry.isRetesselated
```

Comment:
FYI, isRetesselated needs to stay for now. There are some additional plans to remove this, but some changes to the booleans are required.

Reply:
well, that one is copied along with everything else... the problematic one was bounding box that needed to not be copied :)

Reply:
I have the PR for the measurements completed, but something happened to Travis CI. So, hold on.

If you cannot wait, then you can make the same changes to your branch. There's only 4 files changed.

Reply:
It is ok, still work to do here, and more testing, I will wait for that PR to test with it.